### PR TITLE
Modernize StatChange playbook kit

### PR DESCRIPTION
**Pull request by leilaweiss**
Modernize StatChange playbook kit

### CODE REVIEW

File: app/pb_kits/playbook/pb_stat_change/stat_change.rb [hunk 1]
```diff
@@ -46,47 +33,26 @@ def icon
...
-        "pb_stat_change/stat_change"
-      end
+      # def display_icon
+      #   if !icon.nil?
+      #     pb_icon = Playbook::PbIcon::Icon.new(icon: icon, fixed_width: true)
+      #     ApplicationController.renderer.render(partial: pb_icon, as: :object)
+      #   else
+      #     ""
+      #   end
+      # end
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_stat_change/stat_change.rb]: Let's remove this if it's not being used.

File: app/pb_kits/playbook/pb_stat_change/stat_change.rb [hunk 2]
```diff
@@ -46,47 +33,26 @@ def icon
...
 
-    private
+      # def display_value
+      #   pb_icon_element = Playbook::PbBody::Body.new(status: status) do
+      #     display_icon +
+      #       value
+      #   end
+      #   ApplicationController.renderer.render(partial: pb_icon_element, as: :object)
+      # end
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_stat_change/stat_change.rb]: Remove if not used

File: app/pb_kits/playbook/pb_stat_change/stat_change.rb (hunk 1 above)
Comment (leilaweiss) [reply app/pb_kits/playbook/pb_stat_change/stat_change.rb]: i removed it! I was still working on it at the time and still had my commented out code to guide me!

File: app/pb_kits/playbook/pb_dashboard_value/docs/_dashboard_value_default.html.erb [hunk 3]
```diff
@@ -1,21 +1,21 @@
 <%= pb_rails("dashboard_value", props: {
   stat_label: {label: "Decreased Value"},
   stat_value: {value: 1428, unit: "appts"},
-  stat_change: {change: "decrease", value: "26.1%"}
+  stat_change: {change: "decrease", value: 26.1 }
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_dashboard_value/docs/_dashboard_value_default.html.erb]: Extra space after value before closing curly.

File: app/pb_kits/playbook/pb_dashboard_value/docs/_dashboard_value_default.html.erb [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
 }) %>
 
 <br><br>
 
 <%= pb_rails("dashboard_value", props: {
   stat_label: {label: "Increased Value"},
   stat_value: {value: 938, unit: "homes"},
-  stat_change: {change: "increase", value: "26.1%"}
+  stat_change: {change: "increase", value: 56.1 }
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_dashboard_value/docs/_dashboard_value_default.html.erb]: Extra space after value before closing curly.

File: app/pb_kits/playbook/pb_dashboard_value/docs/_dashboard_value_default.html.erb [hunk 5]
```diff
@@ -1,21 +1,21 @@
...
 }) %>
 
 <br><br>
 
 <%= pb_rails("dashboard_value", props: {
   stat_label: {label: "Neutral Value"},
   stat_value: {value: 261, unit: "windows"},
-  stat_change: {value: "26.1%"}
+  stat_change: {value: 86 }
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_dashboard_value/docs/_dashboard_value_default.html.erb]: Extra space after value before closing curly.

File: app/pb_kits/playbook/pb_dashboard_value/docs/_dashboard_value_align.html.erb [hunk 6]
```diff
@@ -10,7 +10,7 @@
   align: "center",
   stat_label: {label: "Top Title Value"},
   stat_value: {value: 1428, unit: "appts"},
-  stat_change: {change: "decrease", value: "26.1%"}
+  stat_change: {change: "decrease", value: 56 }
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_dashboard_value/docs/_dashboard_value_align.html.erb]: Extra space after value before closing curly.

File: app/pb_kits/playbook/pb_dashboard_value/docs/_dashboard_value_align.html.erb [hunk 7]
```diff
@@ -19,5 +19,5 @@
   align: "right",
   stat_label: {label: "Top Title Value"},
   stat_value: {value: 1428, unit: "appts"},
-  stat_change: {change: "decrease", value: "26.1%"}
+  stat_change: {change: "decrease", value: 86.1 }
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_dashboard_value/docs/_dashboard_value_align.html.erb]: Extra space after value before closing curly.

**Pull request merged**